test/suites/standard1/configuration.robot

```diff
@@ -35,6 +49,30 @@ Debug Log Level Produces No Warning
     Setup With Debug Log Level
     Pattern Should Not Appear In Log Output    ${CURSOR}    Unrecognized log level "debug", defaulting to "Normal"
 
+Unknown Audit Log Profile Should Fail
```

Comment:
Is this covered with https://github.com/openshift/microshift/pull/3105/files#diff-8f642c21bb21f507564f8c19ec55c9c83fa24f6578fd9dd26dcf12bf49aabee0R27 ?
Saying it because of time savings for including some e2e tests as unit tests, if possible.